tools/cal_scripts/CTD/ctd_cal_parser.py

```diff
@@ -93,23 +93,23 @@ def main():
         for row in reader:
             lookup[row['serial']] = row['uid']
 
-    os.chdir('Manufacturer Cal Files')
+    os.chdir('manufacturer_cal_files')
```

Comment:
keep in mind the directory structure above. We are already in a calibration directory, and files is just redundant (I know a directory contains files). So `manufacturer` is simple. Also keep in mind that some OS systems do not handle long file paths well.